### PR TITLE
[#175915737] Add markdown editor for cta metadata field

### PR DESCRIPTION
**Pull request by infantesimone**
Schermata 2020-11-30 alle 13 00 34

### CODE REVIEW

**Comment by gunzip** [comment]
I'm not sure this would be useful due to the nature of the cta field (which afaik is mostly structured frontmatter) cc @Undermaken @CrisTofani

**Comment by AleDore** [comment]
> I'm not sure this would be useful due to the nature of the cta field (which afaik is mostly structured frontmatter) cc @Undermaken @CrisTofani

Maybe markdown is too much as format, but it can be useful also as a textarea instead of an input type text, due to parsing problems.

**Comment by infantesimone** [comment]
converted the field to textarea 👍

File: src/components/input/MetadataInput.tsx [hunk 1]
```diff
@@ -40,8 +42,8 @@ const MetadataInput = ({
   t
 }: Props) => {
   return isApiAdmin ? (
-    /* - Input text: all metadata except 'scope' and 'descrition'
-     * - Text area: 'descrition' according to MetadataDescritionEditor
+    /* - Input text: all metadata except 'scope', 'descrition' and 'cta'
+     * - Text area: 'descrition' and 'cta' according to MarkdownEditor
```
Comment (AleDore) [inline src/components/input/MetadataInput.tsx:46]: now cta is not a "markdown" field :)

**Review by AleDore — COMMENTED** [review]
LGTM apart from a little change in comments

**Review by gunzip — APPROVED** [review]

File: src/components/input/MetadataInput.tsx (hunk 1 above)
Comment (infantesimone) [reply src/components/input/MetadataInput.tsx:46]: 👍

**Review by AleDore — APPROVED** [review]

**Pull request merged**